lib/src/block_syntaxes/list_syntax.dart

```diff
@@ -165,7 +155,7 @@ abstract class ListSyntax extends BlockSyntax {
         }
         // End the current list item and start a new one.
         endItem();
-        childLines.add('$checkBoxIndicator$restWhitespace$content');
+        childLines.add(parseTastListItem(restWhitespace + content));
```

Comment:
would 
  `childLines.add(parseTastListItem('$restWhitespace$content'));`
be preferred by dart style guidelines [prefer_interpolation_to_compose_strings](https://dart-lang.github.io/linter/lints/prefer_interpolation_to_compose_strings.html)

Reply:
Thanks! Fixed.